client/web/src/enterprise/batches/settings/BatchChangesSiteConfigSettingsArea.tsx

```diff
@@ -0,0 +1,31 @@
+import InfoCircleIcon from 'mdi-react/InfoCircleIcon'
+import React from 'react'
+import { RouteComponentProps } from 'react-router'
+import { PageTitle } from '../../../components/PageTitle'
+import { queryGlobalBatchChangesCodeHosts } from './backend'
+import { CodeHostConnections } from './CodeHostConnections'
+
+export interface BatchChangesSiteConfigSettingsAreaProps extends Pick<RouteComponentProps, 'history' | 'location'> {
+    queryGlobalBatchChangesCodeHosts?: typeof queryGlobalBatchChangesCodeHosts
+}
+
+/** The page area for all batch changes settings. It's shown in the site admin settings sidebar. */
+export const BatchChangesSiteConfigSettingsArea: React.FunctionComponent<BatchChangesSiteConfigSettingsAreaProps> = props => (
+    <div className="web-content">
+        <PageTitle title="Batch changes settings" />
+        <CodeHostConnections
+            headerLine={
+                <>
+                    <div className="alert alert-info">
+                        <InfoCircleIcon className="icon-inline mr-2" />
+                        You are configuring a <strong>global service account</strong>. The credentials on this page can
```

Comment:
Another wording nitpick from me: do users actually know what we mean with "global service account"? I'm not sure. Maybe we can simply call it what it is - global credentials?

```suggestion
                        You are configuring <strong>global credentials</strong>. The credentials on this page can
```